### ISSUE TITLE
Color module causes collision with Color object

### ISSUE THREAD

**Issue by joetheday** [Microsoft/windows-framer-toolkit#3]
If you have a prototype that uses custom colors like this

```
red = new Color("rgba(255,25,25,1)")
```
and try to use the Color module like this:
```
{Color} = require "Color"
```
Framer will throw an error like this:
```
Object doesn't support this action
```

Need to rename our Color module. Maybe something like `msColor`?

**Comment by LucasHaines** [comment]
is this referring to use theme color? If so, maybe name it systemColor? systemThemeColor? Or something similar to the public XAML resource name for reference?

**Comment by joetheday** [comment]
Good call. We should name it what XAML calls it.

**Comment by joetheday** [comment]
Fixed with pull #6
